lib/hq/graphql/input_object.rb

```diff
@@ -60,6 +60,12 @@ def self.to_graphql
         super
       end
 
+      def self.extends(input_object)
+        input_object.arguments.each do |arg, properties|
+          argument arg.to_sym, properties.graphql_definition.type, required: properties.type.non_null?
+        end
+      end
+
```

Comment:
An argument supports more properties than field name, type, and required. An input object also supports more than `argument`. I'm not sure this is going to be flexible enough. 

Does graphql-ruby offer any solution similar to inputs + implements? That would be the preferred path if we need to update hq-graphql. I had previously asked what our options were, but I never got a response. Is this the only option?

Reply:
Sorry for not responding to the previous question. I have found 2 ways: 
1) Extending an input class: The problem with this is that only 1 class can be extended.
2) Using a nested argument of `LinkedUserInput` type, for example. The downside of this is that we would be adding another level of deepness to the input object.
I will dig more into `graphql-ruby` to see if I can use something from there to not to touch `hq-graphql`, and also will take care of the flexibility issue.

Reply:
Can we use [concerns](https://api.rubyonrails.org/classes/ActiveSupport/Concern.html) for now until this becomes a bigger problem?

```ruby
module Types
  module LinkedUserInput
    extend ActiveSupport::Concern

    included do
      argument :form_linked_user_id, ID, required: false
      argument :is_linked_user_form_update, String, required: false
    end
  end
end
```

Reply:
Ok, I did not know about `Concerns`. Thanks. Will close this PR and use [this](https://github.com/OneHQ/agencieshq/pull/13773/commits/f6df59bbc9fbc512e7d9cf3841d9b9df2835fe31) instead

Reply:
We rarely use concerns as it's just a wrapper around basic ruby. I would invest some time learning Ruby. Learning "mixins" is a good place to start when learning how to reuse logic https://www.tutorialspoint.com/ruby/ruby_modules.htm